source/_static/calendar.csv

```diff
@@ -1,8 +1,8 @@
 Date,Lecture / Activity,Assignments / Reading
 ,**Week 1**,
-M 1/5,Class Intro,
+M 1/5,:slides:ref:`00_intro`,
 W 1/7,History & Background,
-F 1/9,History & Background,
+F 1/9,History & Background,Chapter 1; Chapter 2 (Up to and including Regular expressions); Chapter 22 SSH Subsection (pg 926-930)
```

Comment:
This is a little hard to read because of how the table is setup with the theme. We do have an assignments page, maybe we can put the detailed description there with a link provided from this page that doesn't get cut off. What do you think?

Otherwise we need to find a way to wrap that table better.